src/envo/envo-edit.owl

```diff
@@ -10671,7 +10671,6 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000395> <http://purl.obolibrar
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000395> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_00000141>))
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000395> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_00000501>))
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000395> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000086> <http://purl.obolibrary.org/obo/PATO_0001689>))
-SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000395> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002353> <http://purl.obolibrary.org/obo/ENVO_01001813>))
```

Comment:
This existential restriction was false according to a detailed review of @balhoff and had to be removed, see #1401